exchanges/bithumb/bithumb.go

```diff
@@ -615,7 +639,9 @@ func (b *Bithumb) SendAuthenticatedHTTPRequest(path string, params url.Values, r
 	params.Set("endpoint", path)
 	payload := params.Encode()
 	hmacPayload := path + string(0) + payload + string(0) + b.Nonce.String()
-	hmac := common.GetHMAC(common.HashSHA512, []byte(hmacPayload), []byte(b.APISecret))
+	hmac := common.GetHMAC(common.HashSHA512,
```

Comment:
Are you using a linter for these all of these line changes? The function declaration is longer than this and is on one line, so I don't understand why this is being separated

Reply:
No I have a spacer so I break it down usually when It goes over a certain length.

Reply:
When an individual code line is > 80 characters, it's good to split to a second line.